### PR TITLE
fix(#3524): enable IT

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Closes: #3524

### CODE REVIEW

File: eo-runtime/src/test/java/integration/CopiedSources.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    /**
+     * Ctor.
+     * @param farea Farea
+     */
+    CopiedSources(final Farea farea) {
+        this.farea = farea;
+    }
+
+    @Override
+    public CopiedSources apply(final String target) throws IOException {
```
Comment (yegor256) [inline eo-runtime/src/test/java/integration/CopiedSources.java:57]: @maxonfjvipon this is exactly what the `.save()` method does (in `Farea`). Why re-implementing it here?

Comment (maxonfjvipon) [reply eo-runtime/src/test/java/integration/CopiedSources.java:57]: @yegor256 it seems that it copies only directory without files in it

Comment (yegor256) [reply eo-runtime/src/test/java/integration/CopiedSources.java:57]: @maxonfjvipon make a ticket to Farea, I will fix it, release a new version

**Comment by maxonfjvipon** [comment]
@yegor256 please check, until you fixed the problem in `Farea`, current may work here

**Comment by maxonfjvipon** [comment]
@yegor256 please check, until you fixed the problem in Farea, current solution may work here

**Comment by yegor256** [comment]
@maxonfjvipon try farea 0.14.1, should work

**Comment by maxonfjvipon** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@maxonfjvipon thanks!

**Comment by 0crat** [comment]
@maxonfjvipon Thanks for your contribution! Here's the breakdown of your +4 points: +4 base points, -16 for no code review, and +16 to ensure a minimum reward. While we appreciate your effort, remember that [code reviews](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) are crucial for quality. Your balance is now [+20](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep the contributions coming, and aim for reviewed, high-quality code to maximize your points!

**Pull request merged**